extensions/interactions/FractionInput/directives/FractionInput.js

```diff
@@ -132,21 +126,19 @@ oppia.directive('oppiaInteractiveFractionInput', [
             }
           };
 
-          $scope.$on(EVENT_PROGRESS_NAV_SUBMITTED, function() {
-            $scope.submitAnswer($scope.answer);
-          });
-
           $scope.isAnswerValid = function() {
+            if (typeof $scope.FractionInputForm === 'undefined') {
```

Comment:
I think you can just compare directly to `undefined`.

But why would it be undefined?

Reply:
When the interaction first loads, the FractionInputForm can be not defined yet. But the progress bar can already exist and triggering the answer validity pipeline.